mpitests/test_apply.py

```diff
@@ -219,6 +219,86 @@ def circuit():
     assert np.allclose(local_state_vector, local_expected_output_cpu, atol=tol, rtol=0)
 
 
+def expval_single_wire_no_param(tol, obs):
+    num_wires = numQubits
+    comm = MPI.COMM_WORLD
+    commSize = comm.Get_size()
+    num_global_wires = commSize.bit_length() - 1
+    num_local_wires = num_wires - num_global_wires
+
+    state_vector = np.zeros(1 << num_wires).astype(np.complex128)
+    local_state_vector = np.zeros(1 << num_local_wires).astype(np.complex128)
+
+    state_vector = create_random_init_state(num_wires)
```

Comment:
Do we need the zero `state_vector` since it is redefined below?
```suggestion
    local_state_vector = np.zeros(1 << num_local_wires).astype(np.complex128)

    state_vector = create_random_init_state(num_wires)
```